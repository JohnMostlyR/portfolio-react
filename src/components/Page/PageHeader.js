import React from 'react';
import styled from 'styled-components';
import HeaderSpeechBubble from '../HeaderSpeechBubble';
import HeaderH2 from '../HeaderH2';

const StyledHeader = styled.header`
  display: flex;
  width: 100%;
  padding-left: 1em;
`;

const PageHeader = (props) => (
  <StyledHeader>
    <HeaderSpeechBubble isLeftHanded={props.isLeftHanded}>
      <HeaderH2>{props.title}</HeaderH2>
    </HeaderSpeechBubble>
  </StyledHeader>
);


export default PageHeader;
