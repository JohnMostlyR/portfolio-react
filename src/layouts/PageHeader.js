import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import HeaderSpeechBubble from '../components/HeaderSpeechBubble';
import H2 from '../components/H2';

const StyledHeader = styled.header`
  display: flex;
  width: 100%;
  padding-left: 1em;
`;

const PageHeader = (props) => (
  <StyledHeader>
    <HeaderSpeechBubble isLeftHanded={props.isLeftHanded}>
      <H2>{props.title}</H2>
    </HeaderSpeechBubble>
  </StyledHeader>
);

PageHeader.propTypes = {
  isLeftHanded: PropTypes.bool,
  title: PropTypes.string,
};

export default PageHeader;
