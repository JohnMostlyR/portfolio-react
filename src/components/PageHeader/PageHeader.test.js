import React from 'react';
import renderer from 'react-test-renderer'

import PageHeader from './index';

describe('PageHeader', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<PageHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});