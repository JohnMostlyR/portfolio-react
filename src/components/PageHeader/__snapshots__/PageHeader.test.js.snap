// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader Should render and match the snapshot 1`] = `
<header
  className="sc-bdVaJa hJvvcP"
>
  <div
    className="sc-htpNat hRIAGI"
  >
    <h2
      className="sc-bwzfXH iinwBk"
    />
  </div>
</header>
`;
